arch/X86/X86Mapping.c

```diff
@@ -2923,7 +2923,7 @@ static struct insn_reg2 insn_regs_intel2[] = {
 
 static struct insn_reg insn_regs_intel_sorted [ARR_SIZE(insn_regs_intel)];
 
-static int regs_cmp(const void *a, const void *b)
+static int CAPSTONE_API regs_cmp(const void *a, const void *b)
```

Comment:
I explained it in the commit log, please check it out. I should have mentioned the note in the PR message.